datafusion/src/physical_plan/aggregates.rs

```diff
@@ -64,7 +64,7 @@ pub enum AggregateFunction {
 impl fmt::Display for AggregateFunction {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         // uppercase of the debug.
-        write!(f, "{}", format!("{:?}", self).to_uppercase())
+        write!(f, "{}", self)
```

Comment:
Isn't this doing infinite recursion, as it will try to use the same method to display `self`?